pkg/api/converterfromadmin.go

```diff
@@ -349,9 +354,27 @@ func mergeConfig(oc *admin.OpenShiftManagedCluster, cs *OpenShiftManagedCluster)
 	if in.GenevaLoggingControlPlaneAccount != nil {
 		out.GenevaLoggingControlPlaneAccount = *in.GenevaLoggingControlPlaneAccount
 	}
+	if in.GenevaLoggingControlPlaneEnvironment != nil {
```

Comment:
your new tests found these were missing?  If so I owe you :beer:

Reply:
@jim-minter yes the test discovered the following were missing:

```
Config.GenevaLoggingControlPlaneEnvironment
Config.GenevaLoggingControlPlaneRegion
```